LTSketchbook/libraries/LTC5556/LTC5556.cpp

```diff
@@ -0,0 +1,232 @@
+/*!
+LTC5556: Dual Programmable Downconverting Mixer with IF DVGA
+
+@verbatim
+
+The LTC5556 dual programmable gain downconverting mixer
+is ideal for diversity and MIMO receivers that require
+precise gain setting. Each channel incorporates an active
+mixer and a digital IF VGA with 15.5dB gain control range.
+The IF gain of each channel is programmed in 0.5dB steps
+through the SPI. A reduced power mode is also available
+for each channel.
+
+@endverbatim
+
+http://www.analog.com/en/products/rf-microwave/mixers/single-double-triple-balanced-mixers/ltc5556.html
+
+Copyright 2018(c) Analog Devices, Inc.
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+ - Redistributions of source code must retain the above copyright
+   notice, this list of conditions and the following disclaimer.
+ - Redistributions in binary form must reproduce the above copyright
+   notice, this list of conditions and the following disclaimer in
+   the documentation and/or other materials provided with the
+   distribution.
+ - Neither the name of Analog Devices, Inc. nor the names of its
+   contributors may be used to endorse or promote products derived
+   from this software without specific prior written permission.
+ - The use of this software may or may not infringe the patent rights
+   of one or more patent holders.  This license does not release you
+   from the requirement that you obtain separate licenses from these
+   patent holders to use this software.
+ - Use of the software either in source or binary form, must be run
+   on or directly connected to an Analog Devices Inc. component.
+
+THIS SOFTWARE IS PROVIDED BY ANALOG DEVICES "AS IS" AND ANY EXPRESS OR
+IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, NON-INFRINGEMENT,
+MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.
+IN NO EVENT SHALL ANALOG DEVICES BE LIABLE FOR ANY DIRECT, INDIRECT,
+INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, INTELLECTUAL PROPERTY RIGHTS, PROCUREMENT OF SUBSTITUTE GOODS OR
+SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+*/
+
+//! @ingroup WIP
```

Comment:
Use RF_Timing group here.

Reply:
I don't think I'm in the RF_timing group. Where can I find a list of the groups to add it to so that I can pick the appropriate one? Is that described anywhere?

Reply:
Here are the groups to pick one from:
Analog_to_Digital_Converters
Digital_to_Analog_Converters
Hot_Swap
Power_Monitors
Power_System_Management
Switching_Regulators
Transceivers
Temperature_Monitors
RF_Timing
BMS
PMBus_SMBus
Example_Designs
User_Contributed

Reply:
I don't think any of those are appropriate, can we make a more generic RF group? Timing refers more to PLLs, I would think